src/store/index.js

```diff
@@ -1,38 +1,14 @@
-import { ReducerRegistry } from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
+import { configureStore } from '@reduxjs/toolkit';
+import composes from './reducers/composes';
```

Comment:
Should we come up with a policy on import ordering or something? Like separate libraries from in-tree imports etc..

Reply:
Yes, that's a very good idea... and we should try to put the policy into ESLint.